tests/test_providers/test_datetime.py

```diff
@@ -45,6 +45,11 @@ def test_bulk_create_datetimes_error(self, _datetime):
         with pytest.raises(ValueError):
             _datetime.bulk_create_datetimes(None, None)
 
+        with pytest.raises(ValueError):
```

Comment:
I don't get it. Why error should raise here if timedelta is positive in this particular case?

Reply:
Oh, I get it. It just incrementing `date_start` by `timedelta(0)` that make no sense.